app/models/trait_bank.rb

```diff
@@ -264,11 +269,13 @@ def key_data(page_id)
           "OPTIONAL MATCH (trait)-[:statistical_method_term]->(statistical_method_term:Term) "\
           "OPTIONAL MATCH (trait)-[:units_term]->(units:Term) "\
           "RETURN trait, predicate, object_term, units, sex_term, lifestage_term, statistical_method_term "\
-          # "ORDER BY predicate.position, LOWER(object_term.name), "\
```

Comment:
These changes have nothing to do with inferred_trait -- my editor yelled at me that the limit clause wasn't being concatenated to the query.